app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
+  display: block;
+  background-color: $grey-medium;
+  padding: $gutter;
+  margin-top: $gutter * 2;
+}
+
+.writing-field {
+  display: inline-block;
+  margin-bottom: 24px;
+
+  label {
+    display: block;
+    margin-bottom: $gutter * 1.5;
+    margin-top: $gutter;
+  }
+
+  .space-for-writing {
+    border: 0;
+    border-bottom: 1px solid black;
+  }
+
+  &.name {
+    width: 100%;
+
+    .space-for-writing {
+      width: 97%;
+    }
+  }
+
+  &.signature, &.date {
```

Comment:
I hate having to name bits like this, it's never easy.

On the one hand I'd say find a simple grid helper/thing and use that or come up with a name like .row and col-50